src/components/Chat/ChatItem.tsx

```diff
@@ -17,6 +17,9 @@ import Box from '../Box/Box'
 import { ComponentSlotStylesPrepared } from '../../themes/types'
 
 export interface ChatItemProps extends UIComponentProps, ChildrenComponentProps {
+  /** Indicates whether the ChatItem is the part of a batch. */
+  grouped?: 'start' | 'middle' | 'end'
```

Comment:
My bad for not linking the issue in the PR.. Here is the issue with the discussion around this: https://github.com/stardust-ui/react/issues/588 We were considering using grouped, consecutive prop or additional component for representing the groups of messages. We decided there to use the grouped prop (which in the issue is only as an boolean, but it turned out that we must have some different words for describing the first, the last message and the middle messages in a batch. The options: start, end and middle were chosen because the start and end are the options we are using across all other components. Please share some additional proposals if you have any :)

Reply:
Will add the attached prop

Reply:
speaking of the design language choices, there are apps that are using `grouped` term to express this concept (Telegram as an example).

Thing that I don't like with attached is the following combination of prop name and `top` value:

```jsx
<ChatItem attached='start' />
```
This is really hard to reason about what does this prop mean for the element, especially when it will occur in the following context:

```jsx
...
<ChatItem attached />
<ChatItem attached='end' />
<ChatItem attached='start' />
<ChatItem />
...
```

Reply:
to me it seems that original version with `grouped` is something that reads better:

```jsx
...
<ChatItem grouped='start' />
<ChatItem grouped />
<ChatItem grouped='end' />
<ChatItem />
<ChatItem grouped='start' /> // ???
<ChatItem />
...
```

Reply:
Let's go with `attached: boolean | 'top' | 'bottom'` for now, it satisfies the need for three different attached ChatItems which is sufficient for now, and in the future once we will add this prop in other component, it will be much easier for the users to learn to use it... We may consider changing this in the future if we receive complains for it. Can we agree with this?